### PR TITLE
Fixed Mismatch with Before and After

### PR DESCRIPTION
**Pull request by nothingalike**

### CODE REVIEW

File: CardanoSharp.Wallet/CardanoSharp.Wallet.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
     <LangVersion>9.0</LangVersion>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Version>1.0.4</Version>
+    <Version>1.0.5</Version>
```
Comment (pszeder) [inline CardanoSharp.Wallet/CardanoSharp.Wallet.csproj:7]: Are you not worried about this being a breaking change?

**Review by pszeder — APPROVED** [review]

File: CardanoSharp.Wallet/CardanoSharp.Wallet.csproj (hunk 1 above)
Comment (nothingalike) [reply CardanoSharp.Wallet/CardanoSharp.Wallet.csproj:7]: this controls the nuget version. nothing more

Comment (pszeder) [reply CardanoSharp.Wallet/CardanoSharp.Wallet.csproj:7]: The consumers will likely not expect a breaking change as only the last value in the version is being changed.

Comment (nothingalike) [reply CardanoSharp.Wallet/CardanoSharp.Wallet.csproj:7]: im not really sure what you mean by all this but the Version is what controls the nuget package version that gets published

Comment (pszeder) [reply CardanoSharp.Wallet/CardanoSharp.Wallet.csproj:7]: [https://docs.microsoft.com/en-us/nuget/concepts/package-versioning](https://docs.microsoft.com/en-us/nuget/concepts/package-versioning)

> Major: Breaking changes
> Minor: New features, but backwards compatible
> Patch: Backwards compatible bug fixes only

Comment (nothingalike) [reply CardanoSharp.Wallet/CardanoSharp.Wallet.csproj:7]: would you not view this more of a bug fix?
image
https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-basics

Comment (nothingalike) [reply CardanoSharp.Wallet/CardanoSharp.Wallet.csproj:7]: just to be clear, i honestly dont mind making the version w/e you think is more appropriate

Comment (pszeder) [reply CardanoSharp.Wallet/CardanoSharp.Wallet.csproj:7]: Well, yes it is a bug fix, but also breaks code that depends on this being incorrect. 

So for example I'll have to update the following:
image

I think if we strictly want to follow the versioning guide it should be a major version increase. But I understand you may have other objectives for major versions and this is a small change with probably very few projects actually depending on the old incorrect implementation.

Comment (nothingalike) [reply CardanoSharp.Wallet/CardanoSharp.Wallet.csproj]: sounds like a good enough reason to me. im competely fine with the call to increment major

**Pull request merged**